.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+# Related Issue
+- issue goes here
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Additional Information
+- any additional information or context
+
+# Checklist
+- [ ] Tests
+- [ ] Changelog Entry
+- [ ] Squash Commits
+- [ ] Documentation
+
+# Screenshots
```

Comment:
Not required. This is a command line tool, screenshots aren't usually very helpful.